src/commands/appSettings/downloadAppSettings.ts

```diff
@@ -59,4 +59,6 @@ export async function downloadAppSettings(node?: IAzureNode): Promise<void> {
             }
         }
     });
+
+    await vscode.window.showTextDocument(await vscode.workspace.openTextDocument(localSettingsUri));
```

Comment:
I'm not a fan of having multiple uses of await in a single expression. await tends to abstract away the fact that a separate context is created for evaluating/resolving that promise. That works fine in a single context. However, it can lead to errors like the ones mentioned here: 
https://dev.to/antogarand/javascript-async-maths-is-hard-5b6i and here : https://www.reddit.com/r/javascript/comments/8lih82/javascript_operator_causing_race_conditions_when/?st=jm13b2wo&sh=8abe1c71

I know this goes against your policy of single use expressions, but you could consider this.